Doc/library/bdb.rst

```diff
@@ -73,6 +74,50 @@ The :mod:`bdb` module also defines two classes:
       Print the output of :meth:`bpformat` to the file *out*, or if it is
       ``None``, to standard output.
 
+   :class:`Breakpoint` instances have the following attributes:
+
+   .. attribute:: file
+
+      File name of the :class:`Breakpoint`.
+
+   .. attribute:: line
+
+      Line number of the :class:`Breakpoint` within
+      :attr:`file <bdb.Breakpoint.file>`.
```

Comment:
Since this is all indented under _class:: Breakpoint_, I think you should be able to just link with :attr:`` `file` ``. (Failing that, there is another shortcut: :attr:`` `~bdb.Breakpoint.file` ``.)